content/components/action-menu.mdx

```diff
@@ -11,41 +11,231 @@ export default ComponentLayout
-</Link>
-</div>
-<div>
-<Heading sx={{fontSize: 3}}>To make one or more selections</Heading>
-<Box as="p">
-
-Action menu can be used for making a selection among a small list of options. The list is usually predefined with system values, but in some cases it can include user-provided data when those are known not to grow into too many items.
 
+### Closing
+
+#### Default behavior
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/18661030/226486482-fc457572-342d-42d3-9da6-fa87ef009cea.png"
+      alt=""
+    />
+    <Caption>
+      Close the menu by clicking on an item, clicking on the trigger, or clicking anywhere outside the menu.
+    </Caption>
+  </div>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/18661030/226486483-e3d658ea-4148-47d6-95ed-5ca90cc55b38.png"
+      alt=""
+    />
+    <Caption>Close the menu by hitting enter on an item, or esc.</Caption>
+  </div>
 </Box>
-</div>
+
+#### Single and multi select
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/18661030/226487362-b66daa5f-40b3-4f73-908c-19a74cabd435.mp4"
+    />
+    <Caption>Single select menu closes upon selection and may update the button label.</Caption>
+  </div>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/18661030/226487367-680395c7-1859-40a0-bbfc-53f37a686662.mp4"
+    />
+    <Caption>Multi select menu closes upon selection and can be re-opened to select more options.</Caption>
+  </div>
 </Box>
 
-## Related components
+### Responsive
 
-- [Action list](/components/action-list)
-- [Select panel](https://primer.style/react/SelectPanel)
+<Box
+  mb={3}
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt=""
+    src="https://user-images.githubusercontent.com/18661030/226487827-ad73d646-dfbc-4794-9627-c346bd6fbaa2.png"
+  />
+  <Box as="p" mt="0">
+    <h4>Narrow viewports</h4>
+    For narrow viewports, ActionMenu should remain anchored to its trigger and stay within the bounds of the viewport.
+  </Box>
+</Box>
+
+## Best practices
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/18661030/226486465-1616fde7-733c-4cbd-b4f8-24b3b113731c.png"
+    />
+    <Caption>Use ActionMenu for quick actions.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/18661030/226486469-677e6998-a700-43d7-8048-2033cb7b3eac.png"
+    />
+    <Caption>For complex interactions, use SelectPanel.</Caption>
```

Comment:
Just for consistency with the next example...

![Screenshot 2023-03-21 at 10 15 58](https://user-images.githubusercontent.com/13340707/226577110-029a2bc8-01c5-405b-8285-77270b166919.png)



```suggestion
    <Caption>For complex interactions, use <a href="https://primer.style/react/SelectPanel">SelectPanel</a>.</Caption>
```